src/databricks/labs/lsql/dashboards.py

```diff
@@ -216,8 +217,8 @@ def __post_init__(self):
         if not self.id:
             self.id = self.path.stem if self.path is not None else ""
 
-    def update(self, other: "TileMetadata") -> None:
-        """Update the tile metadata with another tile metadata.
+    def merge(self, other: "TileMetadata") -> "TileMetadata":
```

Comment:
@nfx : Refactored this method also to an implementation that is stateless

Reply:
yep, the right way